docs/compose-file.md

```diff
@@ -475,6 +475,19 @@ Networks to join, referencing entries under the
       - some-network
       - other-network
 
+#### aliases
+
+Alias names for this service on the specified network.
```

Comment:
Maybe add something about these aliases being usable as the hostname to connect to the service? I'm not sure if "name" is enough information here.

Something like this maybe:

> Aliases for this service on the network. Other servers can use this name to connect to this service.

@londoncalling what do you think?